tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Sink to Cloud Storage
+Summary: Learn how to create a changefeed to stream data from a TiDB Dedicated cluster to cloud storage, such as Amazon S3 etc.
+---
+
+# Sink to Cloud Storage
+
+This document describes how to create a changefeed to stream data from TiDB Cloud to cloud storage. Currently, only Amazon S3 is supported.
+
+> **Note:**
+>
+> To stream data to cloud storage, make sure that your TiDB cluster version is v7.1.0 or later.
```

Comment:
```suggestion
> To stream data to cloud storage, make sure that your TiDB cluster version is v7.1.0 or later.  If you want to upgrade cluster to v7.1.0 or higher, [contact PingCAP](https://docs.pingcap.com/tidb/v5.2/mailto:info@pingcap.com) for support.
```

Reply:
@benmaoer https://docs.pingcap.com/tidb/v5.2/mailto:info@pingcap.com is 404. Please update the link